nautilus_core/data/src/aggregation.rs

```diff
@@ -341,7 +341,7 @@ impl BarAggregator for VolumeBarAggregator {
     #[allow(unused_assignments)] // Temp for development
     fn update(&mut self, price: Price, size: Quantity, ts_event: UnixNanos) {
         let mut raw_size_update = size.raw;
-        let raw_step = (self.core.bar_type.spec.step as f64 * FIXED_SCALAR) as u64;
+        let raw_step = (self.core.bar_type.spec().step as f64 * FIXED_SCALAR) as u64;
```

Comment:
It might be good to `let spec = ...` at the top of the function? (like you did elsewhere.)